pytest/tests/sanity/single_shard_tracking.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
```

Comment:
I think you should also need to add this to the sanity test list here:
https://github.com/near/nearcore/blob/master/nightly/pytest-sanity.txt